src/runtime/virtcontainers/mount_linux.go

```diff
@@ -180,7 +180,7 @@ func bindUnmountAllRootfs(ctx context.Context, sharedDir string, sandbox *Sandbo
 			mountLogger().WithField("container", c.id).Warnf("container dir is a symlink, malicious guest?")
 			continue
 		}
-		c.unmountHostMounts(ctx)
+		_ = c.unmountHostMounts(ctx)
```

Comment:
Does this require handling and propagation to the caller?

Reply:
Agreed, good question.